### DISCUSSION TITLE
Beta Update to ImageSharp v3

### DISCUSSION THREAD

**Discussion by stefannikolei** [SixLabors/ImageSharp.Drawing#267] [General] [answered]
Do you plan to release a new Beta in the near future build against ImageSharp v3?

**Comment by JimBobSquarePants** [comment, marked as answer]
We’re gonna ship a v1 targeting ImageSharp v2 first. 

Hopefully once #265 is merged we won’t actually have much more to do to make that happen.

**Reply by JimBobSquarePants** [reply]
Shipped
